http/client/client.go

```diff
@@ -25,13 +26,12 @@ import (
 )
 
 const (
-	// 20 seconds max.
-	requestTimeout = 20
-
 	// 15MB max.
 	maxBodySize = 1024 * 1024 * 15
 )
 
+var cfg = config.NewConfig()
```

Comment:
Hum.. I should refactor this config package. This is going to parse the config values twice during the process startup.

Reply:
I thought about it when I wrote this.

It seem the config never change after the process startup. So it is OK to work though not so good. For now I cannot find out a way without parsing the config value again. Looking forward to your good news